modules/search-service-impl/src/main/java/org/opencastproject/search/impl/solr/SolrIndexManager.java

```diff
@@ -213,6 +213,49 @@ public void clear() throws SolrServerException {
     }
   }
 
+  /**
+   * Removes the entry with the given <code>id</code> from the database.
+   *
+   * @param seriesid
+   *          identifier of the series or episode to delete
+   * @param deletionDate
+   *          the deletion date
+   * @throws SolrServerException
+   *           if an errors occurs while talking to solr
+   */
+  public boolean deleteSeries(String seriesid, Date deletionDate) throws SolrServerException {
+    try {
+      QueryResponse solrResponse = null;
+      try {
+        SolrQuery query = new SolrQuery(Schema.DC_IS_PART_OF + ":" + ClientUtils.escapeQueryChars(seriesid) + " AND -"
+                + Schema.OC_DELETED + ":[* TO *]");
+        solrResponse = solrServer.query(query);
+      } catch (Exception e1) {
+        throw new SolrServerException(e1);
+      }
+
+      if (solrResponse.getResults().size() == 0) {
+        logger.warn("Trying to delete non-existing media package {} from the search index", seriesid);
+        return false;
+      }
+
+      // Use all existing fields
+      SolrDocument doc = solrResponse.getResults().get(0);
```

Comment:
I think the document you get back here represents an event and you throw away all but the first event. I don't think this will work für series with multiple events, will it?

Reply:
i can take a look at this. 
As it is now this case could not be, because the rest-endpoint does not allow to delete a series wich still contains episodes. So if it comes to this code, there is only one series(?episode?) Element left.

Reply:
Yes good catch @lkiesow  now it searches for the right id.